### PR TITLE
feat: add setupFilesAfterEnv support

### PR DESCRIPTION
**Pull request by Mayank1791989**
This PR will add support of `setupFilesAfterEnv`, we already support setupTestFrameworkScriptFile. Jest  deprecated `setupTestFrameworkScriptFile` in favour of `setupFilesAfterEnv`. Read more here https://jestjs.io/docs/en/configuration#setupfilesafterenv-array & https://github.com/facebook/jest/pull/7119

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: src/runner/runESLint.js [hunk 1]
```diff
@@ -30,11 +30,18 @@ const getCachedValues = (config, extraOptions) => {
 const runESLint = ({ testPath, config, extraOptions }) => {
   const start = Date.now();
 
+  // Jest <= 25.x
   if (config.setupTestFrameworkScriptFile) {
     // eslint-disable-next-line import/no-dynamic-require,global-require
     require(config.setupTestFrameworkScriptFile);
   }
 
+  // Jest >= 26.x
```
Comment (SimenB) [inline src/runner/runESLint.js:39]: It was added in 24

Comment (Mayank1791989) [reply src/runner/runESLint.js]: As in peerDeps we have 25 & 26 and both supports `setupFilesAfterEnv`, so remove the comments.

**Comment by Mayank1791989** [comment]
@SimenB Can you check the PR once.

**Comment by SimenB** [comment]
https://github.com/jest-community/jest-runner-eslint/releases/tag/v0.11.1

sorry about the delay 🙂

**Pull request merged**